app/views/user_mailer/mail_new_event.html.erb

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+    <style>
+      /* Email styles need to be inline */
+    </style>
+  </head>
+
+  <body>
+    <p>Hallo <%= @user.name %>,</p>
+    <p>Er is een nieuwe activiteit genaamd '<%= @event.title %>' aangemaakt.</p>
+    <p>Schrijf je zo snel mogelijk in dan wel uit op https://www.zondersikkel.nl/events/<%= @event.id %></p>
+    <p>We hopen je daar te zien!</p>
+    <p>Details van de activiteit:</p>
+    <p>Titel: <%= @event.title %></p>
+    <p>Datum en tijd: begint op <%= @event.date.strftime("%d-%m-%Y") %> om <%= @event.date.strftime("%H:%M") %><% unless @event.end_time.nil? %> en eindigt op <%= @event.end_time.strftime("%d-%m-%Y") %> om <%= @event.end_time.strftime("%H:%M") %><% end %></p>
+    <p><% unless @event.location.nil? %>Locatie: <%= @event.location %><% end %></p>
```

Comment:
Hier mag de <p> in de conditional, anders is er een lege paragraph in de mail.

Reply:
En in plaats van nil? mag je .blank? checken als hij dan leeg is word hij ook weggelaten dus bij `""`